### PR TITLE
Add sriov related functions

### PR DESCRIPTION
**Pull request by Yingshun**
It's required by sriov migration test for now.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
suggest change python name to sriov_utils.py to keep consistent with other function utils

File: virttest/sriov.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+from virttest import remote
+from virttest.staging import service
+
+
+def find_pf(driver, session=None):
+    """
+    Check if available PF exists to test
+
+    :param session: The session object to the host
+    :raise: test.cancel when no pf found
```
Comment (chunfuwen) [inline virttest/sriov.py:20]: generally, we don't have test.cancel available, but just exception in utils module

File: virttest/sriov.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+from virttest import utils_net
+from virttest import remote
+from virttest.staging import service
+
+
+def find_pf(driver, session=None):
+    """
+    Check if available PF exists to test
+
+    :param session: The session object to the host
```
Comment (chunfuwen) [inline virttest/sriov.py:19]: miss driver python doc

File: virttest/sriov.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    :param pci_addr: The pci address
+    :param vf_no: The value to be set
+    :param session: The session object to the host
+    :return: True if successful
+    """
+    logging.debug("pci_addr is %s", pci_addr)
+    cmd = "echo %s > %s/sriov_numvfs" % (vf_no, pci_addr)
+    s, o = utils_misc.cmd_status_output(cmd, shell=True,
+                                        verbose=True, session=session)
+    if not s:
```
Comment (chunfuwen) [inline virttest/sriov.py:62]: generally , we need keep one function or method to have one return value in whatever situation if you want them return value. 
So suggest change to :
return not s

**Comment by Yingshun** [comment]
> suggest change python name to sriov_utils.py to keep consistent with other function utils

Updated the file to utils_sriov.py and fixed all your comments.

**Pull request merged**